core/src/main/resources/error/error-classes.json

```diff
@@ -356,6 +356,11 @@
     ],
     "sqlState" : "22023"
   },
+  "FIELD_NOT_FOUND" : {
```

Comment:
I'm wondering if we should unify column and field (which is nested column) in the error message. cc @srielau

Reply:
Well, that was kind of my earlier question. How is this different from UNRESOLVED_COLUMN? (Which does include fields in the message). But it seems we KNOW that we are talking about a field here (DDL?)

Reply:
>  But it seems we KNOW that we are talking about a field here (DDL?)

Even more specifically, we look for a field in a struct, and the struct doesn't have such field.